### DISCUSSION TITLE
New Fields UI

### DISCUSSION THREAD

**Discussion by jimafisk** [pocketbase/pocketbase#2318] [Polls]
The [v0.14.0 release](https://github.com/pocketbase/pocketbase/releases/tag/v0.14.0) introduced a new Fields UI in the admin dashboard:

> Simplified the collections fields UI to allow easier and quicker scaffolding of the data schema.

<details>
<summary>Previous Field UI</summary>

![previous_field_ui](https://user-images.githubusercontent.com/5913244/232834306-c97893fa-6e7f-48b5-b2aa-1efc8218622f.png)

</details>

<details>
<summary>Current Field UI</summary>

![current_field_ui](https://user-images.githubusercontent.com/5913244/232834353-12e3de97-1c01-4660-ab1f-bf0f72c34dca.png)

</details>

The new UI feels a little "busy" to me and I prefer the self-contained configuration options to the floating pop-up for options. Maybe that's just me? Would love to hear thoughts from other folks. Thanks!

**Comment by ganigeorgiev** [comment]
Hm, my main idea with the fields redesign was to show the required options directly without extra clicks and minimal vertical space "shift":

![fields_screenshot](https://i.imgur.com/39CjKTC.png)

I've added it in my todo to consider experimenting with a combined approach, aka. keep the required options visible but within expandable "accordion" box instead of the popover.

**Reply by jimafisk** [reply]
I see where you're coming from, and I might just be splitting hairs over my personal preference. Either way it's still functional, I just prefer clear grouping vs quick access. Thanks @ganigeorgiev!

**Reply by ganigeorgiev** [reply]
I've pushed in the `develop` branch a small change to the schema fields UI with slightly less cluttered fields grouping and replaced the popover with a sliding panel. The fields list will look something like:

![fields_ui](https://i.imgur.com/Ysb82Lc.png)

_Note that it is still experimental and may change in the final v0.16.0 (there are no ETAs yet)_

**Reply by jimafisk** [reply]
That looks like an improvement to me :+1: 

Trust your gut, the community seems split based on the initial poll results anyway. Thank you Gani!

**Comment by hungcrush** [comment]
![CleanShot 2023-04-27 at 11 54 34@2x](https://user-images.githubusercontent.com/8098916/234762877-52e54f45-2fe1-427b-8717-d2e3dc15b810.png)
I'm sorry, but I prefer the old look :(((( I feel a clear division is more comfortable

**Reply by ganigeorgiev** [reply]
Please try the latest changes from the [`develop` branch](https://github.com/pocketbase/pocketbase/tree/develop). I think it is better than the old and the `master` version.

_btw. note that the colors in your screenshots are slightly lighter/brighter than what we actually use. Do you have custom styles or it is a browser inconsistency/extension/"night mode"?_

**Reply by hungcrush** [reply]
It looks better now.. Thank you.

--
Yes. I have a few customizations for my Media Library feature.. so the CSS has a little conflict
